readme.md

```diff
@@ -2,6 +2,9 @@
 
 Test if an [ES module](https://nodejs.org/api/esm.html) is run directly with Node.js.  Acts as a replacement for `require.main`.
 
+> [!NOTE]
+> If you are using Node.js versions never than `22.18.0` or `24.2.0`, you might want to use the built-in [`import.meta.main`](https://nodejs.org/docs/latest/api/esm.html#importmetamain) instead of this module.
```

Comment:
```suggestion
> If you are using Node.js versions newer than `22.18.0` or `24.2.0`, you should use the built-in [`import.meta.main`](https://nodejs.org/docs/latest/api/esm.html#importmetamain) instead of this module.
```